test/export/test_torchbind.py

```diff
@@ -392,6 +403,52 @@ def forward(self, arg0_1, attr, arg1_1):
     return (getitem_3, add_1)""",  # noqa: B950
         )
 
+    def test_make_fx_tensor_queue_methods(self):
+        class Model(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(3, 2)
+
+            def forward(self, tq, x):
+                tq.push(x.cos())
+                tq.push(x.sin())
+                x_cos = tq.pop() + tq.size()
+                x_sin = tq.pop() - tq.size()
+                return x_sin, x_cos, tq
+
+        mod = Model()
+        tq = torch.classes._TorchScriptTesting._TensorQueue(
+            torch.empty(
+                0,
+            ).fill_(-1)
+        )
+        tq1 = torch.classes._TorchScriptTesting._TensorQueue(
+            torch.empty(
+                0,
+            ).fill_(-1)
+        )
+        x = torch.ones(2, 3)
+        with torch._higher_order_ops.torchbind.enable_torchbind_tracing():
```

Comment:
Why do we need a context manager to enable the tracing?

Reply:
In this particular example, we don't need to this actually. This is some copy-paste typo. Previously, it's aot_export_module where we need to patch the recognized types using enable_torchbind_tracing(). For make_fx, there's no reason to have it here I think.